packages/apollos-data-connector-rock/src/features/__tests__/index.js

```diff
@@ -125,7 +125,7 @@ describe('features', () => {
         PrayerRequest: {
           byDailyPrayerFeed: () => ({
             top: () => ({
-              get: () => ({}),
+              get: () => ({ then: () => null }),
```

Comment:
another option. more than fine either way 

```suggestion
              get: async () => null,
```

or 

```suggestion
              get: () => Promise.resolve(null),
```

Reply:
oh, nice. that's just good for my own understanding of JS promises.